src/SchemaDiff.php

```diff
@@ -59,6 +59,9 @@ private function initStyles()
     public function diff(SchemaInfo $schema1, SchemaInfo $schema2): bool
     {
         $differences = false;
+
+        $differences = $this->compareDatabaseInfo($schema1, $schema2) || $differences;
```

Comment:
Most of this code talks about the database as the 'schema', and the 'schema info', but this method name is comparing 'database info'. Seems like this should be consistent (unless there's a distinction between the terms 'schema' and 'database' here?)

Reply:
Thanks. I renamed from Database to Schema, but missed this one!

Reply:
(I also don't like the mix of attribute/info, but keeping consistent with table info!)